java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -3,22 +3,31 @@
 
 import glide.api.models.BaseTransaction;
 import glide.api.models.commands.SetOptions;
+import java.util.Map;
 import java.util.UUID;
 
 public class TestUtilities {
+    private static final String key1 = "{key}" + UUID.randomUUID();
+    private static final String key2 = "{key}" + UUID.randomUUID();
+    private static final String value1 = "{value}" + UUID.randomUUID();
+    private static final String value2 = "{value}" + UUID.randomUUID();
 
     public static BaseTransaction transactionTest(BaseTransaction baseTransaction) {
-        String key1 = "{key}" + UUID.randomUUID();
-        String key2 = "{key}" + UUID.randomUUID();
+        baseTransaction.set(key1, value1);
+        baseTransaction.get(key1);
 
-        baseTransaction.set(key1, "bar");
-        baseTransaction.set(key2, "baz", SetOptions.builder().returnOldValue(true).build());
+        baseTransaction.set(key2, value2, SetOptions.builder().returnOldValue(true).build());
         baseTransaction.customCommand("MGET", key1, key2);
 
+        baseTransaction.mset(Map.of(key1, value1, key2, value2));
```

Comment:
change to:
`baseTransaction.mset(Map.of(key1, value2, key2, value1));`

so we can assure that the values were changed